src/components/Admin/ConfirmationDialog.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+export default function AlertDialog({ isOpen, onClose, onDelete, user }) {
```

Comment:
where it is used?

Reply:
Export name was old, and haven't any impact on main import. Changed to normal name.